clients/go/apis/v1alpha1/config.go

```diff
@@ -17,7 +17,7 @@ package v1alpha1
 import (
 	"fmt"
 
-	"go.uber.org/multierr"
+	"github.com/hashicorp/go-multierror"
```

Comment:
I needed to switch because we need a way to check if the multierror is "nil" before wrapping.